scripts/upload_git.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+#
+# Upload binary artifacts when a new release is made.
+#
+
+set -ex
```

Comment:
Perhaps we should turn of the trace `-x` to prevent leaking our gh tokens 😉

Reply:
Other than that, this LGTM! Thanks @wenisman! You are amazing!

Reply:
'x' option removed. :)